Dockerfile

```diff
@@ -29,14 +29,15 @@ COPY ${REMOTE_SOURCE}/apis ./apis
 COPY ${REMOTE_SOURCE}/controllers ./controllers
 COPY ${REMOTE_SOURCE}/internal ./internal
 
+USER 0
 RUN make build
 
 
 #@follow_tag(registry-proxy.engineering.redhat.com/rh-osbs/openshift-ose-cli:v4.8)
-FROM registry.ci.openshift.org/ocp/4.8:cli AS origincli
+FROM quay.io/openshift/origin-cli:4.8 AS origincli
```

Comment:
suggest to use 
```
registry.redhat.io/openshift4/ose-cli:v4.10 
```

Reply:
We know for certain these images are public?

Reply:
this one requires agreement to subscription: ```Before downloading or using this Container, you must agree to the Red Hat subscription agreement located at [redhat.com/licenses](https://www.redhat.com/en/about/agreements). If you do not agree with these terms, do not download or```

Reply:
how do you get this error?

Reply:
This is not an error but the verbiage displayed on the catalog UI